packages/components/src/components/bal-pagination/bal-pagination.tsx

```diff
@@ -38,8 +39,21 @@ export class Pagination {
    */
   @Prop() pageRange = 2
 
+  /**
+   * List of tabs names for 'tabs' interface
+   */
   @Prop() tabsNames: string[] = []
 
+  /**
+   * If 'true, the pagination will be sticky to the top
+   */
+  @Prop() sticky = false
+
+  /**
+   * If sticky, the top position will be determined by this value
+   */
+  @Prop() top = 0
```

Comment:
- [ ] add a watcher to this prop. so when it changes we update the this.el.style.top as well

Reply:
done